### PR TITLE
[Independent-Publisher] v1 Jira Connector

### PR DESCRIPTION
**Pull request by Flow-Joe**
Extends the existing Jira connector. Added apiDefinition, apiProperties and Readme

---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verfied by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1001 @@
...
+                        }
+                      },
+                      "description": "cause"
+                    },
+                    "extraData": {
+                      "type": "string",
+                      "description": "extraData"
+                    }
+                  },
+                  "description": "historyMetadata"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:228]: @Flow-Joe Can you please fix the grammatical mistakes of description values.

Documentation - [Description values of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the description values in swagger

File: independent-publisher-connectors/Atlassian Jira/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set JIRA Address",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('yourInstance').atlassian.net/rest/api/3/"
+        }
+      }
+    ],
+    "publisher": "Flow Joe"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Atlassian Jira/apiProperties.json:42]: @Flow-Joe Please add `stackOwner` property as well.

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@Flow-Joe I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1001 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Jira Cloud Content",
```
[the remark this answers is not in the document]
Comment (Flow-Joe) [reply independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json]: This has been changed. Original Jira Cloud Content was a temporary title.

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1001 @@
...
+    "description": "Jira Cloud platform REST API Custom Connector",
```
[the remark this answers is not in the document]
Comment (Flow-Joe) [reply independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json]: Updated description to include additional information offered by the connector.

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1001 @@
...
+    "contact": {
+      "name": "Hitachi Solutions",
+      "email": "jlindstrom@hitachisolutions.com"
```
[the remark this answers is not in the document]
Comment (Flow-Joe) [reply independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json]: Added URL.

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json (hunk 1 above)
Comment (Flow-Joe) [reply independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json]: Fixed grammatical errors.

File: independent-publisher-connectors/Atlassian Jira/apiProperties.json (hunk 2 above)
Comment (Flow-Joe) [reply independent-publisher-connectors/Atlassian Jira/apiProperties.json]: Added stackOwner

**Comment by Flow-Joe** [comment]
All changes have been fixed/made and submitted

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,1002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Jira Connector",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:5]: @Flow-Joe Connector title should not contain a word called "Connector"

Documentation - [Adding title](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by Flow-Joe — COMMENTED** [review]
I believe all changes have been completed

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,1002 @@
...
+                "name": {
+                  "type": "string",
+                  "description": "name"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description"
+                }
+              },
+              "description": "projectCategory"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:915]: @Flow-Joe Can you please provide description values as description values should be free from grammatical and spelling mistakes.

Comment (sriyen-msft) [reply independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:915]: @Flow-Joe Applies to all the similar properties

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

File: independent-publisher-connectors/Atlassian Jira/apiProperties.json [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+      }
+    },
+    "iconBrandColor": "#0052CC",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set JIRA Address",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('yourInstance').atlassian.net/rest/api/3/"
```
Comment (mamurshe) [inline independent-publisher-connectors/Atlassian Jira/apiProperties.json:38]: This connection parameter isn't defined. Add a new parameter called `jirainstance` and replace `yourinstance`.
```
    "connectionParameters": {
      "jirainstance": {
        "type": "string",
        "uiDefinition": {
          "displayName": "JIRA instance",
          "description": "The url where your JIRA instance is hosted (must support https). ",
          "tooltip": "Provide your JIRA domain (eg. myjirainstance.atlassian.net or mycompany.com:443/jira)",
          "constraints": {
            "required": "true"
          }
        }
      },
      "username": {
```

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,1002 @@
...
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/rest/api/3/issue/{issueIdOrKey}": {
```
Comment (mamurshe) [inline independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:25]: Remove `/rest/api/3` from ALL actions. It should look like following.
```
  "paths": {
    "/issue/{issueIdOrKey}": {
      "put": {
        "description": "Edits an issue. A transition may be applied and issue properties updated as part of the edit.",
```

File: independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,1002 @@
...
+    "title": "Jira",
+    "description": "This Jira Connector extends the current offering by Microsoft, increasing the API endpoints offered by the Jira Cloud platform REST API. This adds Issues, Tasks, Projects, Project Categories and User endpoints.",
+    "contact": {
+      "name": "Joe Unwin",
+      "url": "https://www.hitachisolutions.com",
+      "email": "junwin@hitachisolutions.com"
+    }
+  },
+  "host": "flowyco.atlassian.net",
+  "basePath": "/",
```
Comment (mamurshe) [inline independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json:14]:
```
  "basePath": "/rest/api/3",
```

**Review by mamurshe — CHANGES_REQUESTED** [review]

**Review by Flow-Joe — COMMENTED** [review]
Changes have been made and submitted

**Review by mamurshe — APPROVED** [review]

**Pull request merged**